### ISSUE TITLE
Problem with updating GSMessages with Carthage under Xcode 8

### ISSUE THREAD

**Issue by lwalukie** [wxxsw/GSMessages#12]
Hello,
when I try to update my project with using Carthage for Swift 3 (under Xcode 8) I always get following error:

`**\* Building scheme "Alamofire iOS" in Alamofire.xcworkspace
2016-09-20 11:19:24.122 xcodebuild[20966:120977] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/VVDocumenter-Xcode.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2016-09-20 11:19:24.124 xcodebuild[20966:120977] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/UncrustifyPlugin.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2016-09-20 11:19:24.124 xcodebuild[20966:120977] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2016-09-20 11:20:19.886 xcodebuild[21362:122725] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/VVDocumenter-Xcode.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2016-09-20 11:20:19.887 xcodebuild[21362:122725] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/UncrustifyPlugin.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2016-09-20 11:20:19.888 xcodebuild[21362:122725] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin' not present in DVTPlugInCompatibilityUUIDs
**\* Building scheme "GSMessages" in GSMessages.xcodeproj
2016-09-20 11:20:51.261 xcodebuild[21431:123291] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/VVDocumenter-Xcode.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2016-09-20 11:20:51.262 xcodebuild[21431:123291] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/UncrustifyPlugin.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2016-09-20 11:20:51.263 xcodebuild[21431:123291] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin' not present in DVTPlugInCompatibilityUUIDs
*\* CLEAN FAILED **

The following build commands failed:
    Check dependencies
(1 failure)
*\* BUILD FAILED **

The following build commands failed:
    Check dependencies
(1 failure)
A shell task (/usr/bin/xcrun xcodebuild -project /Users/admin/Documents/Projects/Volvo/DealerLocatoriOSGit/Network/Carthage/Checkouts/GSMessages/GSMessages.xcodeproj -scheme GSMessages -configuration Release -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:
2016-09-20 11:20:51.261 xcodebuild[21431:123291] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/VVDocumenter-Xcode.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2016-09-20 11:20:51.262 xcodebuild[21431:123291] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/UncrustifyPlugin.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2016-09-20 11:20:51.263 xcodebuild[21431:123291] [MT] PluginLoading: Required plug-in compatibility UUID 8A66E736-A720-4B3C-92F1-33D9962C69DF for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin' not present in DVTPlugInCompatibilityUUIDs
*\* CLEAN FAILED **

The following build commands failed:
    Check dependencies
(1 failure)
*\* BUILD FAILED **

The following build commands failed:
    Check dependencies
(1 failure)`

Do you know what can cause the issue? I believe that this message 

> GSMessages -configuration Release -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:

somehow indicates the issue, but it tells nothing specific. Do you know if GSMessage can cause the issue?

**Comment by wxxsw** [comment]
I also got this problem, but now I still have no idea, I will continue to solve this problem.

**Comment by lwalukie** [comment]
I think I have solved this - with using carthage need to directly indicate master branch (this is the one which has support for Xcode8 and Swift 3).

**Comment by wxxsw** [comment]
`github "wxxsw/GSMessages"` is work.

The first reason is that I have updated the `GSMessagesExample.xcodeproj` project settings for Swift3, but no update `GSMessages.xcodeproj`.

Then I put commits to `master`, but `github "wxxsw/GSMessages"` still doesn't work. See your reply, I realized that `github "wxxsw/GSMessages"` will use the last version of the `tag`, rather than `master`, thank you for your reply!
